src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs

```diff
@@ -10,7 +10,7 @@ public IEnumerable<Materials> GetAllMaterials()
             return GetMaterialsFromDatabase();
         }
 
-        private IEnumerable<Materials> GetMaterialsFromDatabase()
+        private static List<Materials> GetMaterialsFromDatabase()
```

Comment:
Why are we switching to List rather than IEnumerable?

Reply:
better for performance

Reply:
List inherits IEnumerable.
we choose what to use depending on what to do.

Reply:
I don't agree its better performance but as discussed if SQ is complaining is wanting this change then we'll have to go with it.